### DISCUSSION TITLE
Can I use rainbowkit in my Existing Angular project ?

### DISCUSSION THREAD

**Discussion by deepakkashyap3013** [rainbow-me/rainbowkit#972] [Q&A]
I have an existing codebase in Angular, there is a thought in my mind can I use the kit in my project. I am finding it difficult to integrate as there is a little resources available.

**Comment by 0xBoom** [comment]
I would like to know this as well!

**Comment by DanielSinclair** [comment]
RainbowKit doesn't directly support Angular, as we are tied to `wagmi` which is React-first. But, rendering a React component that wraps the `RainbowKitProvider` and `WagmiConfig` within Angular should be achievable. There are more native approaches to this that will require you to implement some kind of bridging to and from your wrapping component to use Wagmi hooks within your broader project. There is also the possibility of migrating your project to the [Astro Islands](https://docs.astro.build/en/concepts/islands/) build system to reduce the overhead of using your components together. If you make progress in either direction, let me know! Am happy to include an example in our repo and create additional documentation.
